static/src/javascripts/projects/common/utils/has-parent.js

```diff
@@ -0,0 +1,21 @@
+define([
+    'qwery'
+], function (
+    qwery
+) {
+
+    function hasParent(selector, context) {
+        while (!context[0].classList.contains(selector)) {
+            if (context[0].tagName === 'HTML') {
+                return false;
+            } else {
+                context = qwery(context[0].parentElement);
```

Comment:
does this need to be `qwery`d? can you not just set `context` to the `parentElement`?